src/components/TabletIcon/TabletIcon.scss

```diff
@@ -0,0 +1,17 @@
+.tablet-icon {
```

Comment:
There is some issue with align (probably due to `line-height` property). It seems the labels don't align by the center of a tablet

![Screen Shot 2023-08-09 at 13 35 01](https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/5b4f3423-4c7f-410d-9405-042c14b78cfe)